installers/sirepo-dev/radiasoft-download.sh

```diff
@@ -40,4 +40,13 @@ sirepo_dev_main() {
     cd sirepo
     npm install
     cd ..
+    sirepo_dev_npm
```

Comment:
Move one line up so you don't need to cd below.

Reply:
09ed5adf527e153ce4b1c312f35d0f1542891da6